<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choice_model</class>
 <widget class="QWidget" name="choice_model">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>961</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_music">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/model/音乐.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_weather">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/model/天气.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_smart">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/model/智能家居.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_camera">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/model/摄像头.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
